packages/experiments/test/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "../dist/test",
+    "types": ["mocha"]
+  },
+  "references": [{ "path": "../src" }]
```

Comment:
Is @tokey/test-kit missing?

Reply:
I works without it....not sure that it is needed

Reply:
yes